frontend/components/Course/InstructorQueuePage/Queue.tsx

```diff
@@ -77,28 +118,88 @@ const Queue = (props: QueueProps) => {
         await mutate(queueId, { active: false });
     };
 
+    const pinInput = (
+        <div className="ui input">
+            <label
+                htmlFor="changePin"
+                style={{
+                    fontWeight: "bold",
+                    padding: "0.5rem 0.5rem 0.5rem 2rem",
```

Comment:
not a big deal at all, just fyi there's shorthand
https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties#:~:text=The%202%2Dvalue%20syntax%3A%20border%2Dwidth%3A%201em%202em%20%E2%80%94%20The%20first%20value%20represents%20the%20vertical%2C%20that%20is%20top%20and%20bottom%2C%20edges%2C%20the%20second%20the%20horizontal%20ones%2C%20that%20is%20the%20left%20and%20right%20ones.